calva/providers/symbol.ts

```diff
@@ -12,15 +19,17 @@ export class DocumentSymbolProvider implements vscode.DocumentSymbolProvider {
     if (this.state.deref().get('connected')) {
       let client = util.getSession(util.getFileType(document));
       const ns = util.getNamespace(document.getText());
-      const vars = await client.nsVars(ns);
-      const items = await Promise.all(vars["ns-vars"].map((v: any) => {
-        return client.info(ns, v);
-      }));
-      return items.map((info: any) => new vscode.SymbolInformation(
+      const aproposResponse = await client.apropos({
```

Comment:
@PEZ I switched to use [apropos](https://cljdoc.org/d/cider/orchard/0.4.0/api/orchard.apropos) and [its middleware](https://github.com/clojure-emacs/cider-nrepl/commit/cf9f3a0c98e239a6fb9ca2740ad665d129ecc17b).
Mind checking the value of `aproposResponse` trying this branch your system?

Reply:
There are no `apropos-matches` in the response when I run this. It is a `done` response... Let's chat on Slack, maybe we can use Hangout or something to figure it out together.

Reply:
Actually, I now see that I have an `appropos-matches`, an empty array, in the response. I'm sorry for misleading you with my previous comment.